src/lib/CollectStore.js

```diff
@@ -154,7 +154,11 @@ export default class CollectStore {
               Object.assign({}, account, newAttributes)
             )
           } else {
-            return this.dispatch(createAccount(account.auth)).then(result => {
+            const auth = account.auth
+            const account_type = konnector.slug // eslint-disable-line camelcase
```

Comment:
To avoid disabling an eslint rule I suggest:
```js
return this.dispatch(
  createAccount({
    auth: account.auth,
    account_type: konnector.slug
  })
)
```
What do you think ?

Reply:
done